iree_compiler_plugin.cmake

```diff
@@ -8,6 +8,9 @@ find_package(Boost REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
 set(IREE_AMD_AIE_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}")
+set(IREE_MLIR_AIR_SOURCE_DIR "${IREE_AMD_AIE_SOURCE_DIR}/third_party/mlir-air/mlir")
```

Comment:
Out of curiosity, why removing these lines from the CMakeLists and adding them here and here: [iree_runtime_plugin.cmake](https://github.com/nod-ai/iree-amd-aie/pull/462/files#diff-c01404c842f5f36da999f4af30e03786a5c294d6b03b2830a5809cd07c74abfa)?

Reply:
Well `IREE_MLIR_AIE_SOURCE_DIR` is now used in a couple of places so I just moved both. I can put this one back since I guess it still remains only used in one place.